src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -54,10 +54,12 @@ class Wav2Vec2Config(PretrainedConfig):
         hidden_act (:obj:`str` or :obj:`function`, `optional`, defaults to :obj:`"gelu"`):
             The non-linear activation function (function or string) in the encoder and pooler. If string,
             :obj:`"gelu"`, :obj:`"relu"`, :obj:`"selu"` and :obj:`"gelu_new"` are supported.
-        hidden_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+        hidden_dropout (:obj:`float`, `optional`, defaults to 0.1):
             The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
```

Comment:
```suggestion
            The dropout probability for all fully connected layers in the embeddings, encoder, and pooler.
```

Reply:
wow my English grammer teacher would not be proud of me that's for sure! Thanks a lot for spotting all those!